EnglishFilter/sections/foreign.txt

```diff
@@ -1385,7 +1385,7 @@ macapp.so#$##m2 { height: 5px!important; }
 !+ PLATFORM(ios, ext_android_cb)
 10co.biz,10co.xyz,10co.co,10co.me,10co.org#@#iframe[width="728"][height="90"]
 ! https://github.com/AdguardTeam/AdguardFilters/issues/35997
-||ecimg.tw/img/xdsys/v3/layout/h24/sign/channel*/h24/3c/ad_*.jpg
+||ecimg.tw/img/xdsys/*/layout/*/sign/channel*/h24/3c/ad_*.jpg
```

Comment:
There is an additional h24. Also 3c please.

||ecimg.tw/img/xdsys/*/layout/*/sign/channel*/ad_*.jpg